guide/Credential service/Credential Schema.md

```diff
@@ -0,0 +1,140 @@
+# Introduction
+Within the SVX Platform, credential schemas are used to form the basis of a verifiable credential. They structure a set of claims about a Holder and determine the logic behind how the claims are used. Without credential schemas, verifiable credentials would not be able to be issued. 
+
+ 
+# What you need before starting
+The schema file (typically a JSON Schema)
+
+
+# Additional considerations
+Each Credential Schema consists of the following mandatory attributes:
+
+* type (e.g. JsonSchemaValidator2018). The specific type definition determines the content of each data schema.
+
+* id (a URI that identifies the schema file)
+
+Credential schema upload is available to Tenant Administrators and must comply with the following rules:
+
+* A plain JSON object.
+
+* The structure is later checked based on the specification: JSON Schema 
+
+All the attributes in the example are required, but values could be different, e.g. URL address of Schema, name, description etc.
+
+ 
+
+### Example credential schema
+
+```bash
+{ 
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "name": "Example",
+  "description": "Example", 
+  "type": "object", 
+  "properties": { 
+
+    "id": { 
+      "type": "string", 
+    } 
+  }, 
+  "required": ["id"], 
+  "additionalProperties": false, 
+}
+```
+## Create schemas
+**Request**
+
+```bash
+POST/schemas
+```
+**Example Value**
+```bash
+{
+  "schema": {
+    "name": "Schema name",
+    "schema_json": {
+      "$schema": "https://json-schema.org/draft/2019-09/schema",
+      "description": "Basic schema description",
+      "name": "Basic schema name",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "name"
+      ],
+      "additionalProperties": false
+    },
+    "organization_ids": [
+      "9322c384-fd8e-4a13-80cd-1cbd1ef95ba8",
+      "986dcaf4-c1ea-4218-b6b4-e4fd95a3c28e"
+    ]
+  }
+}
+```
+**Responses**
+
+```bash
+201 - Schema created
```

Comment:
I would also add example responses e.g.
`{
  "schema": {
    "id": "7dcfa413-bff0-4f62-a51a-ab9959623332",
    "name": "Schema name",
    "version": "1.0.0",
    "schema_json": {},
    "organization_ids": [
      "9322c384-fd8e-4a13-80cd-1cbd1ef95ba8",
      "986dcaf4-c1ea-4218-b6b4-e4fd95a3c28e"
    ],
    "created_at": "2022-11-24T06:04:47.370Z"
  }
}`

Reply:
We'll add example responses later. First version will be mostly "functional". The Swagger docs will contain that anyway.